packages/toolkit/src/view/model/ModelHubSettingPageMainView.tsx

```diff
@@ -152,7 +198,7 @@ export const ModelHubSettingPageMainView = (
       <ModelConfigurationFields
         model={model.isSuccess ? model.data : null}
         marginBottom="mb-10"
```

Comment:
@orangecoloured If you think we will never use this code, we can safely remove them